pkg/controllermanager/controller/shoot/maintenance/reconciler.go

```diff
@@ -224,6 +224,10 @@ func (r *Reconciler) reconcile(ctx context.Context, log logr.Logger, shoot *gard
 		}
 	}
 
+	// TODO(MichaelEischer): Kubernetes.Kubelet.SystemReserved is no longer supported in Gardener starting from K8S 1.31.
```

Comment:
```suggestion
	// TODO(ialidzhikov): Kubernetes.Kubelet.SystemReserved is no longer supported in Gardener starting from K8S 1.31.
```

ref https://github.com/gardener/gardener/issues/10286#issuecomment-2275673091

Reply:
I've updated the comment.